### ISSUE TITLE
NTR: Spleen biopsy: granulomas (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9121]
New term request Spleen biopsy: granulomas (elthat_100427182618): Splenic  UMLS: 1535984

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000981
- label: Splenic granulomatosis
- parents: [Granulomatosis (HP:0002955)](https://hpo.jax.org/app/browse/term/HP:0002955) | [Abnormal spleen morphology (HP:0025408)](https://hpo.jax.org/app/browse/term/HP:0025408)
- pmid: 
- def: Splenic inflammation involving a granulomatous response, i.e., a non-specific inflammatory response involving granulomas, defined as a compact organized collection of mature mononuclear phagocytes including epithelioid and giant cells.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9121
- synonyms:
